src/store/consts/utils.ts

```diff
@@ -587,7 +587,7 @@ export const handleSimulate = async (
         ticks: ticks,
         tickmap: tickmaps[pool.tickmap.toString()]
       })
-      if (swapSimulateResult.amountPerTick.length >= 8) {
+      if (swapSimulateResult.amountPerTick.length > 19) {
```

Comment:
It will be better to use this `TICK_CROSSES_PER_IX`

Reply:
It's not exported